src/gui/windows/AboutWindow.cpp

```diff
@@ -149,17 +149,17 @@ void AboutWindow::AboutComponent::setupLabels()
     const std::initializer_list<Initialiser> initList
         = {{&appNameLabel,
             juce::String(ProjectInfo::projectName),
-            50,
+            static_cast<float>(50),
             juce::Justification::bottomLeft,
             juce::Colour(225, 225, 225)},
            {&versionLabel,
             juce::String("Version ") + ProjectInfo::versionString,
-            18,
+            static_cast<float>(18),
             juce::Justification::topLeft,
             juce::Colour(180, 180, 180)},
            {&commitHashLabel,
             juce::String(GIT_COMMIT_HASH),
-            10,
+            static_cast<float>(10),
```

Comment:
Same thing here again for all these, use `50.f` instead of `static_cast<float>(50)`.